apps/vr-tests/package.json

```
@@ -22,6 +22,7 @@
     "@fluentui/react-avatar": "^9.0.0-alpha.0",
     "@fluentui/react-button": "^9.0.0-alpha.0",
     "@fluentui/react-cards": "^1.0.0-beta.58",
+    "@fluentui/react-divider": "^0.1.0",
```

```suggestion
    "@fluentui/react-divider": "^9.0.0-alpha.0",
```

For converged components we are using "9.0.0-alpha.0" version, it should be also bumped there 🐱

---

updated